django/core/checks/caches.py

```diff
@@ -53,3 +54,24 @@ def check_cache_location_not_exposed(app_configs, **kwargs):
                 id='caches.W002',
             ))
     return errors
+
+@register(Tags.caches)
```

Comment:
I think it should be a `deploy` check.

Reply:
I've changed it to a deploy check because it's not really an issue in a local environment.

Reply:
but OK let's have it.